content/2019/11/25/shape-editor.rst

```diff
@@ -0,0 +1,41 @@
+public: yes
+author: miriam
+headline:
+  - tagline: For quick visual adjustments directly on the page
+tags: [CSS, 'Mozilla Developer', DevTools, Code, Video]
+image:
+  - src: 'mozdev/shape_path.png'
+summary: |
+  Have you ever wanted to create more interesting shapes on the web,
+  or flow text around the details of an image?
+  Shape paths can be hard to code without a visual reference,
+  but Firefox provides a shape editor to make it quick and clear.
+
+
+Edit your CSS Shapes with the Shape Path Editor
+===============================================
+
+CSS provides a number of built-in shapes we can use --
+from inset boxes to circles, ellipses, and polygons.
+I'll show you how to use the tool to inspect, edit,
+and create shapes directly in the browser.
+
+.. callmacro:: content.macros.j2#video
+  :embed: '<iframe width="560" height="315" src="https://www.youtube.com/embed/u9bDe3Bw0sA" frameborder="0" allow="accelerometer; autoplay; encrypted-media; gyroscope; picture-in-picture" allowfullscreen></iframe>'
+
+  Use the shape path editor to create and adjust shapes on the fly!
+
+Resources:
+
+- `MDN Article » <https://developer.mozilla.org/en-US/docs/Tools/Page_Inspector/How_to/Edit_CSS_shapes>`_
```

Comment:
I'm curious about the `»` here -- not a problem, but not a pattern I see elsewhere.